crates/map/src/helpers/heap.cairo

```diff
@@ -87,6 +90,14 @@ pub trait HeapTrait<T> {
     /// # Effects
     /// * The items are swapped
     fn swap(ref self: Heap<T>, lhs: u8, rhs: u8);
+    /// Print the heap.
+    /// # Arguments
+    /// * `self` - The heap to print
+    /// # Effects
+    /// * The heap is printed
+    /// # Note
+    /// * This function is only for debugging purposes
+    fn print(ref self: Heap<T>);
```

Comment:
This function could be relocated into the printer module to only be added for testing. Or do you need this to be used in katana?

Reply:
I agree with you that it is not consistent with MapPrinter, but I feel like the Type itself should be responsible of how it should render, so it makes sense to me to have it at the type definition location.
If you want to use it for a pure cairo purpose it seems ok to have it directly in the type, if it is for a Starknet purpose it will fail at the compilation time, right?

Reply:
Sounds good to me, that's a good vision on that point, thanks for sharing that.
Yes, now sozo displays a warning, before it wasn't. So it makes it easy anyway to see that you've a print in production code.